python/dgl/contrib/sampling/dis_sampler.py

```diff
@@ -0,0 +1,88 @@
+# This file contains distributed samplers.
+from ...node_flow import NodeFlow
+from ...network import _create_sender, _create_receiver
+from ...network import _send_subgraph, _recv_subgraph
+from ...network import _finalize_sender, _finalize_receiver
+
+class Sender(object):
```

Comment:
Rename it to `SamplerSender` caz it is not a general-purpose sender in RPC.

Reply:
Solved.